client/src/app/game-details/game-details.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {GameProjection} from "../services/group.service";
+import {GameService} from "../services/game.service";
+import {ActivatedRoute} from "@angular/router";
+import {UserProjection} from "../services/dto/user.dto";
+
+@Component({
+  selector: 'app-game-details',
+  templateUrl: './game-details.component.html',
+  styleUrls: ['./game-details.component.sass']
+})
+export class GameDetailsComponent implements OnInit {
+
+  @Input() game: GameProjection;
+
+  @Input() groupId: number;
+
+  constructor(private gameService: GameService, private route: ActivatedRoute) {
+    this.groupId = +route.snapshot.paramMap.get('groupId');
+  }
+
+  async removePlayer(player: UserProjection) {
+    return this.gameService.removePlayer(+this.route.snapshot.paramMap.get('groupId'), this.game.id, player.id).toPromise()
```

Comment:
Please use `this.groupId` here. You've already fetched it from the router.

Reply:
good point